oda_api/api.py

```diff
@@ -419,6 +419,7 @@ def returned_analysis_parameters_consistency(self):
         mismatching_parameters = []
         for k in self.parameters_dict.keys():
             # these do not correspond to meaning
+            # TODO can it be removed from here too? This function to be used only within a test
```

Comment:
maybe let's add comment to remove with reference to the relevant discussion.
And keep for now, why not. It's safe.